regression/goto-analyzer/constant_propagation_01/test-vsd.desc

```diff
@@ -3,7 +3,7 @@ main.c
 --variable-sensitivity --simplify out.gb
 ^EXIT=0$
 ^SIGNAL=0$
-^Simplified:  assert: 1, assume: 0, goto: 1, assigns: 2, function calls: 0$
-^Unmodified:  assert: 0, assume: 0, goto: 0, assigns: 9, function calls: 2$
+^Simplified:  assert: 1, assume: 0, goto: 1, assigns: 1, function calls: 0$
+^Unmodified:  assert: 0, assume: 0, goto: 0, assigns: 10, function calls: 2$
```

Comment:
I don't immediately see why this results in a simplification being skipped.  Do you have any intuition for what is going on here? Is this actually a VSD bug?

Reply:
The commit titled "Properly type initial value of __CPROVER_max_malloc_size" removes a type cast that the simplifier was able to take care of.